<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComponentCheck</class>
 <widget class="QWidget" name="ComponentCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Checking the correcness of a series of component compare to a printer</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Select printer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Checking</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseFileButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse file</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
